### ISSUE TITLE
Is there any way to have IMAGE_ANALYSIS and VIDEO_CAPTURE use cases at same time?

### ISSUE THREAD

**Issue by developer--** [android/camera-samples#349]
I'm using CameraX and want to have imageAnalyzer and video recording at same time.
I'm interested if it can be done with CameraX

**Comment by darrinps** [comment]
Yes. https://stackoverflow.com/questions/57485050/how-to-increase-frame-rate-with-android-camerax-imageanalysis
